src/twisted/web/http.py

```diff
@@ -1784,7 +1784,7 @@ class HTTPChannel(basic.LineReceiver, policies.TimeoutMixin):
     A receiver for HTTP requests.
 
     The L{HTTPChannel} provides L{interfaces.ITransport} and
-    L{interfaces.IConsumer} to the L{Request} objects it creates. It also
+    L{interfaces.IConsumer} to the L{Request} objects it creates.  It also
```

Comment:
Please pardon the irrelevant whitespace changes here; this was *almost* a https://github.com/glyph/python-docstring-mode -compliant docstring, and I re-wrapped it; it's like someone went through and just deleted the double-spacing after it had been already reflowed.  (Maybe an earlier version had a bug?)